index.html

```diff
@@ -723,7 +723,10 @@ <h2>3. We wish you good voting!</h2>
                 next({ path: '/cr-cr', replace: true });
             } else next();
         })
-        Vue.filter('dateCZ', function(date) { return moment(date).format('D. M. YYYY'); })
+        Vue.filter('dateCZ', function(dateString) {
+            const date = moment(dateString);
```

Comment:
bude `const` podporovaný tím core.js polyfillem? Aby nám tohle zas nerozbilo IE…

Reply:
To určitě ne, core.js neřeší language constructs.

Každopádně se používá na několika místech o pár řádků výš a "The const keyword is not supported in Internet Explorer 10 or earlier."

Reply:
👍